### PR TITLE
A bit of top-level docs for CallLogging & DefaultHeaders features

### PR DESCRIPTION
**Pull request by elizarov**
(they are used in HelloApplication sample)

### CODE REVIEW

File: ktor-server/ktor-server-core/src/io/ktor/features/DefaultHeaders.kt [hunk 1]
```diff
@@ -15,8 +19,15 @@ class DefaultHeaders(config: Configuration) {
...
     @Volatile private var cachedDateText: String = ZonedDateTime.now(ZoneOffset.UTC).toHttpDateString()
 
+    /**
+     * Configuration for [DefaultHeaders] feature.
+     */
     class Configuration {
         val headers = ValuesMapBuilder()
+
+        /**
+         * Adds standard header property [name] with the specified[value]
```
Comment (orangy) [inline ktor-server/ktor-server-core/src/io/ktor/features/DefaultHeaders.kt]: Space before `[value]`

Comment (orangy) [reply ktor-server/ktor-server-core/src/io/ktor/features/DefaultHeaders.kt]: Also inconsistent use of _period_ at the end of the text, other docs in this PR have them. I wonder what should be the global style on this. Should we end all API docs with period or not?

Comment (elizarov) [reply ktor-server/ktor-server-core/src/io/ktor/features/DefaultHeaders.kt]: Yes. IMHO, we should use period.

**Comment by elizarov** [comment]
Updated

**Comment by orangy** [comment]
Now, on the commit message. See https://chris.beams.io/posts/git-commit/ for the overall idea about how we try (but not always follow) to phrase commit messages. It should be something like "Document CallLogging & DefaultHeaders features".

**Pull request merged**